package.js

```diff
@@ -2,11 +2,11 @@ Package.describe({
   name: 'ostrio:flow-router-extra',
   summary: 'Carefully extended flow-router with waitOn and template context',
   version: '3.10.1',
-  git: 'https://github.com/veliovgroup/flow-router'
+  git: 'https://github.com/veliovgroup/flow-router',
 });
 
 Package.onUse((api) => {
-  api.versionsFrom(['1.4', '3.0-beta.0']);
+  api.versionsFrom(['3.0']);
   api.use(['modules', 'ecmascript', 'promise', 'tracker', 'reactive-dict', 'reactive-var', 'ejson', 'check'], ['client', 'server']);
 
   api.use(['zodern:types@1.0.13', 'typescript'], ['client', 'server'], { weak: true });
```

Comment:
In the line below this one (now shown in this diff) you need to support Blaze 2 and 3:

```js
  api.use(['templating', 'blaze@2.0.0 || 3.0.0'], 'client', { weak: true });
```